### PR TITLE
change info about touchscreen

### PR DESCRIPTION
**Pull request by ShalokShalom**
This adds a lot of details

I am on the European version, NixOS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,15 +46,15 @@ Tested disctributions, kernel versions:
 | Ports | 3 × USB-C, Mini-jack | ✔ Yes | All the ports work. Charging works only via left port. |
 | Graphic Dongle | USB-Typ-C to USB-Typ-A-/HDMI-/VGA | ✔ Yes | Works. HDMI monitor works, USB keyboard works. |
 | Keyboard |  | ⚠️ Minor issues | see [Keyboard details](#keyboard) below |
-| Touchpad | | ✔ Yes | Touchpad is detected and works good in GNOME. Left, right clicks, 2-finger scrolling, 2-finger zooming, 3-finger workspaces switching work. |
+| Touchpad | | ⚠️ Only the US version | EU/Yoga variant has no touchpad. Otherwise, all features work on the US variant. |
```
Comment (milkovsky) [inline README.md]: Do you mean a touch screen? I have bought my in Europe, Austria and I have both touchscreen and touchpad working.

**Review by milkovsky — CHANGES_REQUESTED** [review]
Thank you for the contribution. There is one open question in my comments.

File: README.md (hunk 1 above)
Comment (ShalokShalom) [reply README.md]: Well, interesting. I also bought mine in Austria and I have never seen a touchscreen mentioned in reviews of the European unit at all. 

inxi -Fxz reports Yoga Slim 7 Carbon 14ACN6 as the model for me. 

No touchscreen for me on both Arch and NixOS. 
The original owner used it on Windows 11, and has also reported, that is has no touchscreen.

Comment (ShalokShalom) [reply README.md]:
```
cat /proc/bus/input/devices
I: Bus=0011 Vendor=0001 Product=0001 Version=ab83
N: Name="AT Translated Set 2 keyboard"
P: Phys=isa0060/serio0/input0
S: Sysfs=/devices/platform/i8042/serio0/input/input0
U: Uniq=
H: Handlers=sysrq kbd leds event0 
B: PROP=0
B: EV=120013
B: KEY=402000000 3803078f800d001 feffffdfffefffff fffffffffffffffe
B: MSC=10
B: LED=7

I: Bus=0019 Vendor=0000 Product=0001 Version=0000
N: Name="Power Button"
P: Phys=PNP0C0C/button/input0
S: Sysfs=/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0C0C:00/input/input1
U: Uniq=
H: Handlers=kbd event1 
B: PROP=0
B: EV=3
B: KEY=10000000000000 0

I: Bus=0019 Vendor=0000 Product=0005 Version=0000
N: Name="Lid Switch"
P: Phys=PNP0C0D/button/input0
S: Sysfs=/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0C0D:00/input/input2
U: Uniq=
H: Handlers=event2 
B: PROP=0
B: EV=21
B: SW=1

I: Bus=0019 Vendor=0000 Product=0006 Version=0000
N: Name="Video Bus"
P: Phys=LNXVIDEO/video/input0
S: Sysfs=/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:0e/LNXVIDEO:00/input/input3
U: Uniq=
H: Handlers=kbd event3 
B: PROP=0
B: EV=3
B: KEY=3e000b00000000 0 0 0

I: Bus=0019 Vendor=0000 Product=0000 Version=0000
N: Name="Ideapad extra buttons"
P: Phys=ideapad/input0
S: Sysfs=/devices/pci0000:00/0000:00:14.3/PNP0C09:00/VPC2004:00/input/input7
U: Uniq=
H: Handlers=kbd event4 rfkill 
B: PROP=0
B: EV=13
B: KEY=400000000000000 0 0 6000800000000000 100000000000 0 81000800100c03 4400000000300400 0 2
B: MSC=10

I: Bus=0018 Vendor=04f3 Product=31ef Version=0100
N: Name="MSFT0001:00 04F3:31EF Mouse"
P: Phys=i2c-MSFT0001:00
S: Sysfs=/devices/platform/AMDI0010:01/i2c-1/i2c-MSFT0001:00/0018:04F3:31EF.0004/input/input8
U: Uniq=
H: Handlers=event5 mouse0 
B: PROP=0
B: EV=17
B: KEY=30000 0 0 0 0
B: REL=3
B: MSC=10

I: Bus=0018 Vendor=04f3 Product=31ef Version=0100
N: Name="MSFT0001:00 04F3:31EF Touchpad"
P: Phys=i2c-MSFT0001:00
S: Sysfs=/devices/platform/AMDI0010:01/i2c-1/i2c-MSFT0001:00/0018:04F3:31EF.0004/input/input10
U: Uniq=
H: Handlers=event6 mouse1 
B: PROP=5
B: EV=1b
B: KEY=e520 10000 0 0 0 0
B: ABS=2e0800000000003
B: MSC=20

I: Bus=0000 Vendor=0000 Product=0000 Version=0000
N: Name="HD-Audio Generic HDMI/DP,pcm=3"
P: Phys=ALSA
S: Sysfs=/devices/pci0000:00/0000:00:08.1/0000:03:00.1/sound/card0/input11
U: Uniq=
H: Handlers=event7 
B: PROP=0
B: EV=21
B: SW=140

I: Bus=0000 Vendor=0000 Product=0000 Version=0000
N: Name="HD-Audio Generic HDMI/DP,pcm=7"
P: Phys=ALSA
S: Sysfs=/devices/pci0000:00/0000:00:08.1/0000:03:00.1/sound/card0/input12
U: Uniq=
H: Handlers=event8 
B: PROP=0
B: EV=21
B: SW=140

I: Bus=0001 Vendor=10ec Product=0287 Version=0001
N: Name="HDA Digital PCBeep"
P: Phys=card1/codec#0/beep0
S: Sysfs=/devices/pci0000:00/0000:00:08.1/0000:03:00.6/sound/card1/input13
U: Uniq=
H: Handlers=kbd event9 
B: PROP=0
B: EV=40001
B: SND=6

I: Bus=0000 Vendor=0000 Product=0000 Version=0000
N: Name="HD-Audio Generic Mic"
P: Phys=ALSA
S: Sysfs=/devices/pci0000:00/0000:00:08.1/0000:03:00.6/sound/card1/input14
U: Uniq=
H: Handlers=event10 
B: PROP=0
B: EV=21
B: SW=10

I: Bus=0000 Vendor=0000 Product=0000 Version=0000
N: Name="HD-Audio Generic Headphone"
P: Phys=ALSA
S: Sysfs=/devices/pci0000:00/0000:00:08.1/0000:03:00.6/sound/card1/input15
U: Uniq=
H: Handlers=event11 
B: PROP=0
B: EV=21
B: SW=4

```

Comment (milkovsky) [reply README.md]: @ShalokShalom I bought mine on `lenovo.com/at`. The touchscreen was an optional upgrade. I checked it on. But do not worry about it. I do not recommend it personally. I do not use it at all, and time after time I accidentally touch it and the cursor disappears for a few seconds.

Comment (ShalokShalom) [reply README.md:49]: Ah, then we can simply list it as such optional upgrade?

Comment (ShalokShalom) [reply README.md]: I guess you could disable it. Afaik, should that also increase battery life slightly.

Comment (milkovsky) [reply README.md]: I would love to deactivate it. I have not found an option in the BIOS for deactivating it.

File: README.md [hunk 2]
```diff
@@ -46,15 +46,15 @@ Tested disctributions, kernel versions:
 | Ports | 3 × USB-C, Mini-jack | ✔ Yes | All the ports work. Charging works only via left port. |
 | Graphic Dongle | USB-Typ-C to USB-Typ-A-/HDMI-/VGA | ✔ Yes | Works. HDMI monitor works, USB keyboard works. |
 | Keyboard |  | ⚠️ Minor issues | see [Keyboard details](#keyboard) below |
-| Touchpad | | ✔ Yes | Touchpad is detected and works good in GNOME. Left, right clicks, 2-finger scrolling, 2-finger zooming, 3-finger workspaces switching work. |
+| Touchpad | | ✔ Yes | Touchpad is an optional purchase. It works as intended and is fully functional. |
```
Comment (milkovsky) [inline README.md]: This line is about touchpad, not the touchscreen. You could add the comment about the "optional purchase" to the "Display " section https://github.com/milkovsky/Linux-on-Lenovo-Slim-7-Carbon-AMD#touchscreen.

Comment (ShalokShalom) [reply README.md]: Oh god, how could I oversee that. Thanks for the heads up *shame* :D

File: README.md (hunk 1 above)
Comment (ShalokShalom) [reply README.md]: Well, you can blacklist the device to the Linux kernel?
I did this once with a wifi chip, I believe. 

I am pretty sure I used /etc/modprobe.d/blacklist.conf. for that, although there are also udev rules and xinput. Considering Wayland is the future, I suggest one of the first two. 

We could also include that in the README, I assume.

File: README.md (hunk 2 above)
Comment (ShalokShalom) [reply README.md]: Considering there are multiple ways to drive a touchscreen, can you confirm which way you use?

That differs from distro to distro. 
https://wiki.archlinux.org/title/Touchscreen

Comment (milkovsky) [reply README.md]: No worries :)
I did not have to do any extra steps to setup touchscreen. It worked out-of-the-box for Ubuntu, PopOs, Fedora in my tests.

**Comment by milkovsky** [comment]
@ShalokShalom thanks a lot! merged

**Comment by milkovsky** [comment]
There is one important open issue here: https://github.com/milkovsky/Linux-on-Lenovo-Slim-7-Carbon-AMD/issues/2
If you have any ideas, feel free to help.

**Comment by milkovsky** [comment]
@ShalokShalom There is one important open [issue about speakers ](https://github.com/milkovsky/Linux-on-Lenovo-Slim-7-Carbon-AMD/issues/2) here.
If you have any ideas, feel free to help. It will be very appreciated.

**Comment by ShalokShalom** [comment]
@milkovsky Yeah, I was already looking at it. I ran into an issue with the software that is supposed to reasign the pins.

**Pull request merged**